src/components/popup/details/purposes/purposes.jsx

```diff
@@ -94,6 +96,42 @@ export default class Purposes extends Component {
 		return () => this.props.onShowVendors(filter);
 	}
 
+	isPublisherPurposeRequireConsent(purposeID) {
+		const { legIntPurposeIds, contractPurposeIds } = config;
+		if (!legIntPurposeIds || !contractPurposeIds) {
```

Comment:
it should works even we define `legIntPurposeIds` or `contractPurposeIds` only.

Reply:
✅